packages/website/docs/Button.md

```diff
@@ -1,39 +1,43 @@
-## Guidelines
+## Best Practices
 
-### General
+Button label should allow users to **foresee what will happen** when clicking them. 
 
--   **To redirect** users to another page of the current website or app, or to another website, **use links** rather than buttons.
--   Avoid multiplying buttons within a given page. If you need several buttons, less popular or **less important actions may be grouped** in a ["more" Menu button](../#/components/Menu) (dropdown list) or visually styled as isolated actions or links.
+Avoid multiplying buttons within a page. A page should have **only one primary action button**. If several secondary action buttons are required, consider using [Actionable items](#/form/ActionableItem) instead.
 
-### Labels
+When using button groups, such as for "Save" and "Cancel" actions, always put the action that move the user forward through his journey **to the right**. For instance, order buttons as "Cancel" then "Save".
 
--   The button label should allow users to **foresee what will happen** when clicking it.
--   Button labels should be **short** and use **trigger words**.
--   As much as possible, the **first word** of the label should be an **action verb** (e.g., the label for a button that allows the user to add a filter could be "Add Filter").
 
-### Alterative Texts
+## Labeling
 
--   When a button includes **only an icon** (i.e., no label), **label information** should be **added in the alternative text** element in the HTML and appear **in a tooltip** when hovering the button.
--   When a button contains a **text as well as an icon**, the icon is a visual help to identify the purpose faster. Providing an **alternative text is not recommended** as it would only repeat the information when using a **screenreader**.
+Keep labels short, preferably **under three words**.
+Use **trigger words** to clearly indicate the action performed by the button.
```

Comment:
what is a trigger word? An action? :thinking:

Reply:
Trigger words are the verbs/action words that will influence a user into clicking. 
They are more specific than regular verbs. Maybe we could call them "Commands" verbs 
@jlevesquecoveo any thoughts on this?

E.g.: 
Using "View Profile" instead of "Open Profile", 
Using "Publish Story" instead of  "Save Story" (depending on the context)
Using "Create X" instead of  "Save" at the end of a stepper modal when we add something in the platform.